utils/text_formatting_links.test.jsx

```diff
@@ -30,7 +30,7 @@ describe('Markdown.Links', () => {
 
         assert.equal(
             Markdown.format('test.:test').trim(),
-            '<p>test.:test</p>'
+            '<p><a class="theme markdown__link" href="test.:test" rel="noreferrer" target="_blank">test.:test</a></p>'
```

Comment:
Could you move this to the `external links` section since we now consider this a link?

Reply:
Done